Sources/PackageModel/Module.swift

```diff
@@ -16,6 +16,7 @@
 */
 
 @_exported import enum PackageDescription.SystemPackageProvider
+import struct Basic.RelativePath
```

Comment:
Oh cool.  That code will end up looking a lot cleaner once the APIs are in terms of paths.